Rich-Text-Editor-(RTE)-Overview.md

```diff
@@ -1,117 +1,137 @@
 
-# Upgrading CKEditor
+### Upgrading CKEditor
+
 As described above, the core of our RTE is the 3rd party library CKEditor. If you are upgrading the CKEditor version we are using, it is important to check that the new CKEditor version does not cause any regressions in the RTE. After upgrading the CKEditor version, use the RTE in the exploration editor to perform the following checks in both the Chrome and Firefox browsers:
-* Input some plaintext and save. Ensure that the content doesn't disappear upon save.
+
+* Input some plain text and save. Ensure that the content doesn't disappear upon save.
 * Input a single Rich Text Component (such as the Math component) and then save. Make sure it is saved properly.
 * Try using backspace to delete the Rich Text Component. Make sure it is actually deleted.
-* Try copy and pasting a Rich Text Component. Hit save and make sure that the component is copied properly.
-* Try drag and dropping a Rich Text Component from one place in the content to another, then saving. Make sure the component is moved properly.
+* Try copying-and-pasting a Rich Text Component. Hit save and make sure that the component is copied properly.
+* Try editing the Rich Text Component and make sure your changes are shown correctly.
+* Try dragging-and-dropping a Rich Text Component from one place in the content to another, then saving. Make sure the component is moved properly.
 * Input text with multiple paragraphs. Make sure the paragraphs are correctly spaced apart.
 * Insert an Image component with a large image, and save. Make sure the image does not overflow the content boundaries.
 * Insert an Image component with a small image, and save. Make sure the image is not stretched out.
 
-# Rich Text Components
-Oppia explorations can have Rich Text Components, which are custom widgets creators can insert into their content. Each component is an Angular directive, of the form `<oppia-noninteractive-*></oppia-noninteractive-*>`, with different attributes depending on the component. Examples include the Math (for inserting Latex) component, the Image component, and the Video component. This section describes how the components are defined in case you want to add your own component, or need to modify an existing one.
+## Rich Text Components
+
+Oppia explorations can have Rich Text Components, which are custom widgets that creators can insert into their content. Note that we have two kinds of components at play here: rich text components and Angular components. Each rich text component is an Angular component of the form `<oppia-noninteractive-*></oppia-noninteractive-*>`, with different attributes depending on the component. Examples include the Math (for inserting Latex) component, the Image component, and the Video component. This section describes how the components are defined in case you want to add your own component, or need to modify an existing one.
```

Comment:
Maybe the first mention of "Angular components" can link to the part of our wiki which describes them.

Reply:
done